### ISSUE TITLE
support using other ports?

### ISSUE THREAD

**Issue by joamatab** [gdsfactory/klive#5]
for developer containers in docker

how about allowing to set the port that it uses?

![image](https://github.com/gdsfactory/klive/assets/4514346/a5f8feb6-30bd-4ef9-97c4-e25598fe8cae)

@sebastian-goeldi 
@skandanC
@tvt173
@Jan-David-Black

**Comment by sebastian-goeldi** [comment]
I have thought about it recently. I think the best way is to add a GUI element that allows you to set additional IPs and the port to listen on. I have to say for me this is low-prio though as I have no intention of using a non-standard port and even less desire to write another GUI dialog in klayout, as those are always a pain to write.

**Comment by jan-david-fischbach** [comment]
I have also been thinking about this: we might as well just check a range of ports for mqtt brokers from the translator side. (and subscribe to all of them?). Like this we can at least avoid problems with a couple of dev containers being connected at the same time, causing the port forwarded to to increment automatically.

**Comment by sebastian-goeldi** [comment]
I don't think this is still really necessary? I will close for now, feel free to open again if really necessary
